gauntlet/packages/gauntlet-solana-contracts/src/core/proto/ocr2Proto.proto

```diff
@@ -0,0 +1,37 @@
+syntax="proto3";
```

Comment:
Where is this file from? It will also be required for the monitoring service AFAIK cc @topliceanu

Let's move this to the root e.g. `./proto/ocr2.proto`

Reply:
From libocr. Monitoring uses the same, but does not include: https://github.com/smartcontractkit/chainlink-solana/pull/26/files#diff-832dbc2305017d852ec2b4160c3b9d4f18443f44fd1696e6d8dced401c949722R31 
which gauntlet needs

Reply:
Yeah, let's just move it to root and monitoring can use directly while Gauntlet derives/generates the `ts` descriptors by using the local script.